Install with Go requires 1.20+

**Description**

Trying to install from Go with 1.19 and the install terminates with this error. And nothing is installed. Upgrading to Go 1.20 fixes it.

```
$  go install github.com/sigstore/cosign/v2/cmd/cosign@latest
go: downloading github.com/sigstore/cosign/v2 v2.2.1
go: downloading github.com/sigstore/cosign v1.13.1
....
go: downloading github.com/dustin/go-humanize v1.0.1
# github.com/buildkite/agent/v3/internal/ordered
go/pkg/mod/github.com/buildkite/agent/v3@v3.58.0/internal/ordered/unmarshal.go:107:15: vdst.Elem().SetZero unde                                           fined (type reflect.Value has no field or method SetZero)
note: module requires Go 1.20
$ go version
go version go1.19.13 linux/amd64
```

Or :
```
$ go/bin/go1.20 install github.com/sigstore/cosign/v2/cmd/cosign@latest

$ ls go/bin/cosign
go/bin/cosign
```

--- a/content/en/system_config/installation.md
+++ b/content/en/system_config/installation.md
@@ -5,9 +5,9 @@ title: Installation
 weight: 950
 ---
 
-## With Go 1.19+
+## With Go 1.20+
 
-If you have Go 1.19+, you can directly install Cosign by downloading the Cosign binary and running:
+If you have Go 1.20+, you can directly install Cosign by running:
 
 ```bash
 go install github.com/sigstore/cosign/v2/cmd/cosign@latest

